flow-nodes/api-builder-plugin-format-date/package-lock.json

```diff
@@ -0,0 +1,1045 @@
+{
```

Comment:
@alasdairhurst , I am not clear about this comment here, could you clarify this, please?

Reply:
I made a similar comment in one of your other PRs... we're not committing package-lock.json files.  These files are automatically generated by npm, and they "should" be added to source control, but we don't because they're a PITA to maintain and noise.  So, it should be added to .gitignore, and not added (accidentally) to source.

Reply:
Yup, removing that file and adding it in gitignore.

Reply:
Resolved.